pkg/core/resources/manager/cache.go

```diff
@@ -88,7 +92,7 @@ func (c *cachedManager) List(ctx context.Context, list model.ResourceList, fs ..
 	if !opts.IsCacheable() {
 		return fmt.Errorf("filter functions are not allowed for cached store")
 	}
-	cacheKey := fmt.Sprintf("LIST:%s:%s", list.GetItemType(), opts.HashCode())
+	cacheKey := fmt.Sprintf("LIST:%s:%s"+HashSuffix(ctx), list.GetItemType(), opts.HashCode())
```

Comment:
Btw I think that instead of  `list.GetItemType` we could retrieve the `res.Descriptor().Name` somehow. Which means that we could potentially just make `opts.HasCode` work well

Reply:
But there could be multiple resources with the same name, so how would that help?